README.md

```diff
@@ -51,6 +51,25 @@ KubeEdge is composed of the following components:
 
 <img src="./docs/images/kubeedge_arch.png">
 
+## To start developing KubeEdge
+
+The [set up] hosts all information about building KubeEdge from source, how to setup. etc.
+
+If you want to build KubeEdge right away there are one options:
```

Comment:
can we rephrase to "To build KubeEdge from source there is one option"

Reply:
done